src/openloco/companymgr.cpp

```diff
@@ -16,6 +22,8 @@ namespace openloco::companymgr
     static loco_global<uint8_t, 0x00525E3C> _byte_525E3C;
     static loco_global<uint8_t, 0x00525E3D> _byte_525E3D;
 
+    static loco_global<int32_t, 0x00e3f0b8> gCurrentRotation;
```

Comment:
Could you make a getter function for this in `viewportmgr.cpp`? It already exists as a `loco_global` there, and I'd prefer to keep the number of `loco_global`s to a minimum.

Reply:
Considering #235, wouldn't it make more sense to have it passed into the function? Or better: access it as a getter on a viewport, so we could actually have different rotations per viewport.

Reply:
Yea if it was something like `viewport->rotation()` then that could later return a different value for each viewport.

Reply:
Something like `viewport->rotation()` sounds perfect, and future proof, yes.